tensorflow/python/framework/config.py

```diff
@@ -237,6 +237,7 @@ def set_optimizer_experimental_options(options):
       - disable_meta_optimizer: Disable the entire meta optimizer.
       - min_graph_nodes: The minimum number of nodes in a graph to optimizer.
         For smaller graphs, optimization is skipped.
+      - Autoparallel optimizer - Automatically parallelizes graphs by splitting along the batch dimension
```

Comment:
This needs to match the Gappler name so it should be auto_parallel - Automatically ....  See https://github.com/tensorflow/tensorflow/blob/26949be08531e29eba6291053b3a679a1663036f/tensorflow/core/grappler/optimizers/meta_optimizer.cc#L247